sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -40,12 +43,57 @@ class HFTransformersForecaster(BaseForecaster):
         Path to the huggingface model to use for forecasting. Currently,
         Informer, Autoformer, and TimeSeriesTransformer are supported.
     fit_strategy : str, default="minimal"
-        Strategy to use for fitting the model. Can be "minimal" or "full"
+        Strategy to use for fitting (fine-tuning) the model. This can be one of
+        the following:
+        - "minimal": Fine-tunes only a small subset of the model parameters,
+          allowing for quick adaptation with limited computational resources.
+        - "full": Fine-tunes all model parameters, which may result in better
+          performance but requires more computational power and time.
+        - "lora": Uses the LoRA (Low-Rank Adaptation) technique to fine-tune the
+          model efficiently with low-rank updates.
+        - "loha": Similar to LoRA, but with higher-rank adaptations.
+        - "adalora": An adaptive version of LoRA that adjusts the rank
+          dynamically during fine-tuning.
     validation_split : float, default=0.2
         Fraction of the data to use for validation
     config : dict, default={}
         Configuration to use for the model. See the `transformers`
         documentation for details.
+    peft_config_dict : dict, default={"target_modules": ["q_proj", "v_proj"]}
```

Comment:
Sorry with coming up with this so late. Would it be possible to just pass the LoraConfig/LohaConfig object from HuggingFace directly. In the code the only adaption would be `get_peft_model(model, peft_config)`. 

This should significantly reduce the maintainance workload for us since all new adaption methods would be directly available if HF releases a new Peft library version.

Reply:
I have made the changes

1. just to confirm we are originally backing out from this - so we have `peft_config` in param instead of `peft_config_dict` - but the user still has control so it should not be an issue

https://github.com/sktime/sktime/pull/6457#discussion_r1616238911
> The user would probably like to have control about the peft_configs. I.e., by providing a dict of parameters that is passed to the LoraConfig.

2. `fit_strategy` can now be one of `["minimal", "full", "peft"]` instead of `["minimal", "full", "lora", "loha", "adalora"]`